packages/core/README.md

```diff
@@ -40,7 +53,27 @@ export const MyApp = () => {
 };
 ```
 
-You can find a list of all available components on our [Storybook page](https://react95.github.io/React95/)
+You can find a list of all available components on our [Storybook page](https://react95.github.io/React95/).
+
+### About ```Modal```
+
+Since ```8.0.0``` the ```Modal``` component is managed with ```ModalContext```.
+So you should surround a group of Modals with a ```<ModalProvider>``` to make
+sure they are managed correctly. Example:
```

Comment:
We can skip the version when talking about `Modal`/`ModalProvider`. There are many more changes and if we want to mention them, we could [reference the release instead](https://github.com/React95/React95/releases/tag/%40react95%2Fcore%408.0.0) (somehow).

```suggestion
The ```Modal``` component is managed with ```ModalContext```.
To ensure they are managed correctly, surround one or a group of Modals with a ```<ModalProvider>```. Example:
```

Reply:
> We can skip the version when talking about `Modal`/`ModalProvider`. There are many more changes and if we want to mention them, we could [reference the release instead](https://github.com/React95/React95/releases/tag/%40react95%2Fcore%408.0.0) (somehow).

Yeah that's clear and simple! Of course it will be better.